setup.py

```diff
@@ -11,7 +11,9 @@
     author='Vincent Driessen',
     author_email='me@nvie.com',
     description=__doc__,
-    packages=find_packages(exclude='tests'),
```

Comment:
Ohhh, I think I know what's wrong: the argument to `exclude=` should be a list. By passing in a string, it'll probably exclude the dirs `'t', 'e', 's', 't', 's'` ;)